scio-google-cloud-platform/src/test/scala/com/spotify/scio/bigquery/types/ConverterProviderSpec.scala

```diff
@@ -51,6 +51,12 @@ final class ConverterProviderSpec
       .retryUntil(_.precision <= Numeric.MaxNumericPrecision)
       .map(Numeric.apply)
   }
+  implicit val arbJson: Arbitrary[Json] = Arbitrary(
+    for {
+      key <- Gen.alphaStr
+      value <- Gen.alphaStr
+    } yield Json("{\"" + key + "\":\"" + value + "\"}")
```

Comment:
Nit: triple quote is nicer when writing json
```suggestion
    } yield Json(s"""{"$key":"$value"}""")
```

Reply:
TIL. I always supposed triple quote was only for multi-line.